paas/briefs-frontend.j2

```diff
@@ -8,7 +8,7 @@
       DM_DATA_API_AUTH_TOKEN: {{ api.auth_tokens[0] }}
       DM_DATA_API_URL: https://dm-api-{{ environment }}.cloudapps.digital
 
-      DM_MANDRILL_API_KEY: {{ shared_tokens.mandrill_key }}
+      DM_NOTIFY_API_KEY: {{ notify_api_key }}
```

Comment:
I'm not decoding credentials to check this, but what's the logic behind moving the token out of the "shared_tokens" section?  Feels like that's exactly what it is.